src/components/ListCard.vue

```diff
@@ -5,7 +5,7 @@
     </div>
     <div>
       <p class="post-date">{{ dateString }}</p>
-      <p class="big align-left">{{ title }}</p>
+      <p class="big align-left">{{ name }}</p>
```

Comment:
change this back to title

it breaks news
title is generalizable, name is not.

Sorry for the confusion here.